docs/src/User_Guide/model_input.md

```diff
@@ -726,3 +739,13 @@ This file contains the settings parameters required to run the Method of Morris
     8. Higher number of num\_trajectory and len_design_mat would lead to higher accuracy
     9. Upper and lower bounds should be specified for all the resources included in the resource `.csv` file (inside the `Resource`). If a parameter related to a particular resource is not uncertain, specify upper bound = lower bound = 0.
 
+### 2.8 Hydrogen\_demand.csv
+
+This file contains inputs specifying regional hydrogen production requirements. This file is needed if `electrolyzer.csv` is included in the resources folder or there are electrolyzer components in `Vre_stor.csv`.
+
+###### Table 25: Structure of the Hydrogen\_demand.csv file
+---
+|**Column Name** | **Description**|
+| :------------ | :-----------|
+|Zone| zone number|
+|Hydrogen_Demand_kt| Hydrogen production requirements for zone *, in 1,000 tons|
```

Comment:
I can't find the `zone` column in the file. Instead, the file contains the following columns:

    H2DemandConstraint
    ConstraintDescription
    Hydrogen_Demand_kt
    PriceH2"